.vscode/README.md

```diff
@@ -22,49 +22,100 @@ Install the following VSCode extensions:
 - Modern Fortran:
   https://marketplace.visualstudio.com/items?itemName=krvajalm.linter-gfortran
 
-Now, configure the VSCode files for the modflow6 directory.
-Open the `modflow6` directory in VSCode.
-Make sure that the setting "python.defaultInterpreterPath" points toward your Python executable.
+- vscode-modern-fortran-formatter:
+  https://marketplace.visualstudio.com/items?itemName=yukiuuh.vscode-modern-fortran-formatter
 
-```json
-{
-    "python.defaultInterpreterPath": "/path/to/python",
-}
-```
+- Remote - WSL:
```

Comment:
Why is WSL a requirement?
@mjr-deltares and me use it just fine on Windows as well?

Reply:
I'm not sure what you mean by requirement- isn't this an informational / help document? It was included as information intended to be helpful for those wishing to use VSCode with WSL.  Also line 31: " Remote - WSL is only needed if you want to use a windows vscode installation in a WSL environment."

If you think this confuses the matter I can remove.

Reply:
I would suggest to add a small section about WSL, but don't mention WSL anywhere else.